notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -106,29 +106,7 @@
    "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    @torch.jit.script_if_tracing
</pre><p>can we move patches for chatglm also out of notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Actually I used to try moving this patch out of notebook, but failed.
Another reason is it may reload some dependencies again in separate file.